languages/en.php

```diff
@@ -477,6 +477,9 @@
 	'admin:cron:period' => 'Cron period',
 	'admin:cron:friendly' => 'Last completed',
 	'admin:cron:date' => 'Date and time',
+	'admin:cron:msg' => 'Message',
+	'admin:cron:started' => 'Cron jobs for "%s" started at %s',
```

Comment:
In the context of the table do we need "Cron jobs for period" in here? Why not "Started (time). May have failed or is still running."